Utilities/DataLoading/EdFi.XmlLookup.Console/log4net.config

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<log4net>
+  <!-- A1 is set to be a ColoredConsoleAppender -->
+  <appender name="A1" type="log4net.Appender.ConsoleAppender">
```

Comment:
Rename the appenders to have more semantic meaning.

Reply:
this is a legacy artifact